wiki/History_of_osu!/en.md

```diff
@@ -1,10 +1,22 @@
 # History of osu!
 
-1. [2007/2008](/wiki/History_of_osu!/1 "2007/2008") - The beginning of osu!, written by [Sinistro](http://osu.ppy.sh/u/Sinistro "Sinistro") (page based from ``/p/history``, with more images).
-2. [2009/2010](/wiki/History_of_osu!/2 "2009/2010") - ?
-3. [2011/2012](/wiki/History_of_osu!/3 "2011/2012") - osu!mania
-4. [2013/2014](/wiki/History_of_osu!/4 "2013/2014") - ?
-5. [2015/2016](/wiki/History_of_osu!/5 "2015/2016") - osu!coins
-6. [2017/2018](/wiki/History_of_osu!/6 "2017/2018") - ?
-
-[History of osu!wiki can be found here instead.](/wiki/History_of_osu!/osu!wiki "osu!wiki history")
+Records of osu!'s history, all in the osu!wiki.
+
+- [2007](/wiki/HOO/2007) - Initial game build
+- [2008](/wiki/HOO/2008) - Taiko and Catch the Beat, more mods, and major improvements
+- [2009](/wiki/HOO/2009) - osu! iPhone
+- 2010
+- [2011](/wiki/HOO/2011) - osu!wiki (MediaWiki)
+- [2012](/wiki/HOO/2012) - osu! is 5, osu!mania, (some) widescreen support, and visual settings
+- [2013](/wiki/HOO/2013) - UI overhaul, HD skins, (rest of) widescreen support, osu!mania editor, in-game collections, moddingV2 was introduced, Disqus, osu!api, accuracy overlays, Comic Fiesta 2013, downloads unrestricted\*, and osu!cuttingedge (osu!supporters only)
+- [2014](/wiki/HOO/2014) - BPM scrolling, in-game intro/outro sequences, osu!cuttingedge (for everyone\*), and snow!
+- [2015](/wiki/HOO/2015) - jizz (redesigned osu!web), osu!weekly, and osu!coins
+- [2016](/wiki/HOO/2016) - scorev2, open-source osu! announced (osu!lazer), and osu!wiki (Markdown/Git)
+- [2017](/wiki/HOO/2017) - first osu!lazer build release, and Discord
+- [2018](/wiki/HOO/2018) - ?
+
+---
+
+On a side note, the osu!wiki has a bit of its own detailed history in and of itself.
```

Comment:
On the other side

Reply:
osu!wiki isn't really the other side of osu!, I had meant for it to be a small thing on its own.